web/modules/custom/dept_migrate/src/MigrateUuidLookupManager.php

```diff
@@ -337,12 +337,18 @@ public function lookupBySourceFileId(array $fids) {
         'd7uuid' => $row->uuid,
         'd7type' => $row->type,
         'd7filename' => $row->filename,
+        'd7uri' => $row->uri,
       ];
     }
 
     // Match up to D9 files using uuid as key from migrate table.
     foreach ($map as $d7fid => $file) {
       $table = 'migrate_map_d7_file';
+
+      if (preg_match('/^private/', $file['d7uri'])) {
+        $table = 'migrate_map_d7_file_private';
```

Comment:
Needs an adjustment to look up the correct table, if we're dealing with a private file